sources/Cargo.lock

```diff
@@ -3573,10 +3798,23 @@ dependencies = [
  "bitflags 1.3.2",
  "cfg-if",
  "libc",
- "memoffset",
+ "memoffset 0.7.1",
  "pin-utils",
 ]
 
+[[package]]
+name = "nix"
+version = "0.30.1"
```

Comment:
Can we update our pinned workspace version of nix so we don't need an extra version?